Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -16,6 +16,13 @@ file that was distributed with this source code.
         {%- if required %} required="required"{% endif %}
     />
 
+    <select id="{{ id }}_autocomplete_input_v4" data-sonata-select2="false"
+        {%- if read_only is defined and read_only %} readonly="readonly"{% endif -%}
+        {%- if disabled %} disabled="disabled"{% endif -%}
+        {%- if required %} required="required"{% endif %}
```

Comment:
@core23 I didn't, but this the goal of this PR is to fix a very bogus problem.

This cleaning stuff can be done later. :wink: